test/e2e_node/image_volume.go

```diff
@@ -77,13 +78,18 @@ var _ = SIGDescribe("ImageVolume", nodefeature.ImageVolume, func() {
 			},
 		}
 
+		if len(nodeName) > 0 {
+			pod.Spec.NodeName = nodeName[0]
+		}
```

Comment:
Does this mean that the rest of node names are ignored? It makes me think that `nodeName ...string` is not the best way to express this parameter. Would `nodeName string` be better?

Reply:
updated.